fauxfactory/__init__.py

```diff
@@ -72,6 +73,29 @@ def _is_positive_int(length):
     if not isinstance(length, int) or length <= 0:
         raise ValueError("{0} is an invalid 'length'.".format(length))
 
+def _unicode_letters_generator():
+    """Generates unicode characters in the letters category
+
+    :return: a generator which will generates all unicode letters available
+
+    """
+    if sys.version_info.major == 2:
+        chr_function = unichr
+        range_function = xrange
+    else:
+        range_function = range
+        chr_function = chr
```

Comment:
Can you order the two variable assignments the same way in both the `if` and `else`? It's a minor thing, but it helps me quickly compare the two blocks of code.